localization/english/dvg_flavor_venice_l_english.yml

```diff
@@ -0,0 +1,45 @@
+﻿l_english:
```

Comment:
Getting some errors in the log:
[localization_reader.cpp:212]: Expected quotation mark (") at end of line 23 in localization/english/dvg_flavor_venice_l_english.yml
[localization_reader.cpp:135]: Expected colon(:) at line 24 and column 4 in localization/english/dvg_flavor_venice_l_english.yml